### PR TITLE
Pass no update 1

### PR DESCRIPTION
**Pull request by hellsworth**
This drops the stdout of updatesnapyaml.py into an output file so that we can check if it exists or not, and then change the output_file name to snapcraft.yaml.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -24,7 +24,8 @@ runs:
     - name: run updatesnapyaml
       id: updatesnapyaml
       run: |
-        ./desktop-snaps/updatesnap/updatesnapyaml.py --github-user $GITHUB_USER --github-token $GITHUB_TOKEN https://github.com/${{ github.repository }} > snapcraft.yaml
+        ./desktop-snaps/updatesnap/updatesnapyaml.py --github-user $GITHUB_USER --github-token $GITHUB_TOKEN https://github.com/${{ github.repository }}
+        mv output_file snapcraft.yaml
```
Comment (sergio-costas) [inline action.yml]: If output_file doesn't exist, mv will fail. You should do:

```suggestion
        if [ -f output_file ]; then
            mv output_file snapcraft.yaml
        fi
```

File: updatesnap/updatesnapyaml.py [hunk 2]
```diff
@@ -107,8 +107,8 @@ def main():
         has_update = True
 
     if has_update:
-        print(manager_yaml.get_yaml())
-        sys.exit(0)
+        with open('temp-file', 'w') as output_file:
```
Comment (sergio-costas) [inline updatesnap/updatesnapyaml.py]: This sends the output to "temp-file", while the action expects "output_file".

File: action.yml (hunk 1 above)
Comment (hellsworth) [reply action.yml]: oh yep, this is needed :)

File: updatesnap/updatesnapyaml.py (hunk 2 above)
Comment (hellsworth) [reply updatesnap/updatesnapyaml.py]: oh oops. An artifact from a previous suggestion.

**Comment by sergio-costas** [comment]
lgtm. Squashing and merging...

**Pull request merged**